fh-messaging/lib/mongo_functions/metrics_r.js

```diff
@@ -2,30 +2,69 @@
 //ts and emit are used for testing purposes
 var ts;
 var emit;
-function metrics_r(k, vals) {
-  // Take the domain and ts in the first vals entry as the reduced elements domain and ts
-  var ret = {}, i, p, tempVal;
-
-  for (i in vals) {
-    if (vals.hasOwnProperty(i)) {
-      tempVal = vals[i];
-      for (p in tempVal) {
+
+function metrics_r(k, metrics) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'metrics_r' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)